source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -45,13 +45,16 @@ and signed in, in different web browser tabs.
 service you want to set up. Select __Settings__, then __Account credentials__,
 then __Edit credentials__.
 
-1. In your Worldpay browser tab, go to the __Profile__ page, then select
-__Identification__. 
+1. In your Worldpay browser tab, select __Account__ in the left-hand
+menu. 
 
-1. Copy the Worldpay __Merchant Code__ into the GOV.UK Pay __Merchant ID__
-   field.
+1. From the __Identification__ section, copy the Worldpay __Merchant Code__ into
+the GOV.UK Pay __Merchant ID__ field.
 
-1. Copy the Worldpay __Original XML Username__ into the GOV.UK Pay __Username__ field.
+1. Select the checkbox __Enable original username__ and select __save__. 
```

Comment:
should the numbering be 3-5?

Reply:
Markdown doesn't care what the numbers are - it'll do the right thing